package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "asciidoctor-vscode",
 	"displayName": "%displayName%",
 	"description": "%description%",
-	"version": "0.0.0",
+	"version": "2.9.0-dev",
```

Comment:
it makes things easier when you want to install the extension locally (otherwise VS Code will complain that version 0.0.0 cannot be installed)